### PR TITLE
Prometheus data exporting

### PR DESCRIPTION
**Pull request by huntears**

### CODE REVIEW

**Comment by huntears** [comment]
Some ideas of data to export:
- Number of chunks loaded
- Number of entities loaded
- MSPT
- TPS

File: cubic-server/Server.cpp [hunk 1]
```diff
@@ -165,32 +156,20 @@ void Server::launch(const configuration::ConfigHandler &config)
 
     _writeThread = std::thread(&Server::_writeLoop, this);
 
+#if PROMETHEUS_SUPPORT == 1
+    if (CONFIG["monitoring-prometheus-enable"].as<bool>()) {
+        _prometheusExporter = std::make_unique<PrometheusExporter>("0.0.0.0:4242");
```
Comment (STMiki) [inline cubic-server/Server.cpp:161]: Is this really the default port for prometheus ? :')

**Review by STMiki — APPROVED** [review]

File: cubic-server/Server.cpp (hunk 1 above)
Comment (Trompettesib) [reply cubic-server/Server.cpp:161]: Can it be 8080?

Comment (huntears) [reply cubic-server/Server.cpp:161]: Bruh I forgot to make it a config xd

Comment (huntears) [reply cubic-server/Server.cpp:161]: Lemme fix that real quick

**Comment by huntears** [comment]
image
It looks like this now

**Review by STMiki — APPROVED** [review]

**Review by Trompettesib — APPROVED** [review]
:+1:

**Pull request merged**